critiquebrainz/frontend/templates/review/modify/base.html

```diff
@@ -108,4 +114,5 @@
       });
     });
   </script>
+  <script src="{{ get_static_path('scripts/rating.js') }}" type="text/javascript"></script>
```

Comment:
JS files shouldn't be accessed directly. The way we do it here now is by building JS files in *gulpfile.js* and referencing them like this: `<script src="{{ get_static_path('rating.js') }}"></script>`. That way you can easily use React, jQuery, etc. in your scripts.